templates/consul.sysv.erb

```diff
@@ -52,7 +52,7 @@ start() {
         mkrundir
         [ -f $PID_FILE ] && rm $PID_FILE
         daemon --user=<%= scope.lookupvar('consul::user') %> \
-            --pidfile="$PID_FILE" \
+            --pidfile="$PID_FILE" --umask=<%= scope.lookupvar('consul::umask') %> \
```

Comment:
Oops, sorry about that. I guess I'll have to just call the shell built in unmask before launching the daemon.